novawallet/Modules/AssetConversion/Service/AssetConversionFlowState.swift

```diff
@@ -101,17 +101,19 @@ final class AssetConversionFlowFacade {
             throw ChainAccountFetchingError.accountNotExists
         }
 
-        let hydra = HydraFlowState(
+        let flowStateStore = HydraFlowStateStore.getShared(
```

Comment:
Didn't get why it is here. It is used only on swaps but we need it in all places where we have fee in custom asset. I believe it should be in the fee registry